client/extensions/woocommerce/state/sites/settings/stripeConnectAccount/test/handlers.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import {
+	createAccount,
+} from '../actions.js';
+import {
+	handleCreateRequest,
+	handleCreateRequestSuccess,
+	handleCreateRequestError,
+} from '../handlers.js';
+import {
+	WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_SUCCESS,
+	WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_ERROR,
+} from 'woocommerce/state/action-types';
+import { WPCOM_HTTP_REQUEST } from 'state/action-types';
+
+describe( 'handlers', () => {
+	describe( '#handleCreateAccountRequest', () => {
+		it( 'should dispatch a POST request', () => {
+			const siteId = '123';
+			const email = 'foo@bar.com';
+			const country = 'US';
+			const dispatch = spy();
+			const action = createAccount( siteId, email, country );
+			handleCreateRequest( { dispatch }, action );
+			expect( dispatch ).to.have.been.calledWithMatch( {
+				type: WPCOM_HTTP_REQUEST,
+				body: JSON.stringify( { email, country } ),
+				method: 'POST',
+				path: `/jetpack-blogs/${ siteId }/rest-api/`,
+				query: {
+					path: '/wc/v1/connect/stripe/account/&_method=POST',
+					json: true,
+					apiVersion: '1.1',
+				}
+			} );
+		} );
+	} );
+
+	describe( '#handleCreateAccountRequestSuccess()', () => {
+		it( 'should dispatch create account receive on success with the account info', () => {
+			const siteId = '123';
+			const email = 'foo@bar.com';
+			const countryCode = 'US';
+			const store = {
+				dispatch: spy(),
+			};
+			const response = {
+				data: {
```

Comment:
Just a note that these responses will change since you are not using `&_envelope`. It should just be the data getting returned, without body and status.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/17922/commits/427f3944f4b8a58d59ac7785e0695175bad9aff0